three.js/src/threex/threex-armarkercontrols-nft.js

```diff
@@ -279,6 +279,10 @@ ARjs.MarkerControls.prototype._initArtoolkit = function () {
         var workerBlobUrl = URL.createObjectURL(workerBlob);
         var worker = new Worker(workerBlobUrl);
 
+        var importUrl = 'artoolkit-nft.min.js';
```

Comment:
This at the end will become a setting in the armarkerControl, so you will be able to set your path for the lib.

Reply:
This is not needed anymore because **artoolkit.min.js** is incorporated inside the BlobWorker itself